src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(addError);
+  }, 3000);
+});
+
+function addSuccess() {
+  document.body.insertAdjacentHTML('beforeend', `
+    <div class="message">
+      Promise was resolved!
```

Comment:
you can pass your message to resolve, reject methods and take them in then/catch

you can make your functions more reusable in this way